.github/workflows/ci.yml

```diff
@@ -10,26 +10,30 @@ jobs:
       - name: Install GCC multilib
         run: |
           sudo apt-get update
-          sudo apt-get install -y gcc-multilib g++-multilib
+          sudo apt-get install -y gcc-multilib g++-multilib ninja-build
```

Comment:
Should this be installing cmake as well?

Reply:
cmake is a preinstalled on the GH actions runner images, and they provide a newer version than you’d get from apt.